app/controllers/bulk_actions/catalog_record_id_and_barcode_csv_jobs_controller.rb

```diff
@@ -1,9 +1,9 @@
 # frozen_string_literal: true
 
 module BulkActions
-  class CatkeyAndBarcodeCsvJobsController < ApplicationController
+  class CatalogRecordIdAndBarcodeCsvJobsController < ApplicationController
     include CreatesBulkActions
-    self.action_type = "SetCatkeysAndBarcodesCsvJob"
+    self.action_type = "SetCatalogRecordIdsAndBarcodesCsvJob"
```

Comment:
Beyond renames, I have *not* yet touched or otherwise tested any of the job-/bulk action-related switching/changes. I'd do this next if the approach in this PR makes sense.

Reply:
Yeah, it naturally followed from deciding the genericized version of `catkey` would be `catalog record id`.